src/debug.cpp

```diff
@@ -87,6 +89,8 @@ std::set<std::string> ignored_messages;
 
 }
 
+
+
```

Comment:
Two empty lines.

Reply:
Damn :( leftovers from a previous try... Thank you!